docs/development/test-vectors.rst

```diff
@@ -37,29 +37,40 @@ Asymmetric ciphers
 Custom Asymmetric Vectors
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ``asymmetric/PEM_Serialization/ec_private_key.pem`` - Contains an Elliptic
+* ``asymmetric/PEM_Serialization/ec_private_key.pem`` and
+  ``asymmetric/DER_Serialization/ec_private_key.der`` - Contains an Elliptic
   Curve key generated by OpenSSL from the curve ``secp256r1``.
-* ``asymmetric/PEM_Serialization/ec_private_key_encrypted.pem`` - Contains the
+* ``asymmetric/PEM_Serialization/ec_private_key_encrypted.pem`` and
+  ``asymmetric/DER_Serialization/ec_private_key_encrypted.der``- Contains the
   same Elliptic Curve key as ``ec_private_key.pem``, except that it is
   encrypted with AES-128 with the password "123456".
-* ``asymmetric/PEM_Serialization/ec_public_key.pem`` - Contains the public key
+* ``asymmetric/PEM_Serialization/ec_public_key.pem`` and
+  ``asymmetric/DER_Serialization/ec_public_key.der``- Contains the public key
   corresponding to ``ec_private_key.pem``, generated using OpenSSL.
 * ``asymmetric/PEM_Serialization/rsa_private_key.pem`` - Contains an RSA 2048
   bit key generated using OpenSSL, protected by the secret "123456" with DES3
   encryption.
-* ``asymmetric/PEM_Serialization/rsa_public_key.pem`` - Contains an RSA 2048
+* ``asymmetric/PEM_Serialization/rsa_public_key.pem`` and
+  ``asymmetric/DER_Serialization/rsa_public_key.der``- Contains an RSA 2048
   bit public generated using OpenSSL from ``rsa_private_key.pem``.
 * ``asymmetric/PEM_Serialization/dsaparam.pem`` - Contains 2048-bit DSA
   parameters generated using OpenSSL; contains no keys.
 * ``asymmetric/PEM_Serialization/dsa_private_key.pem`` - Contains a DSA 2048
   bit key generated using OpenSSL from the parameters in ``dsaparam.pem``,
   protected by the secret "123456" with DES3 encryption.
-* ``asymmetric/PEM_Serialization/dsa_public_key.pem`` - Contains a DSA 2048 bit
+* ``asymmetric/PEM_Serialization/dsa_public_key.pem`` and
+  ``asymmetric/DER_Serialization/dsa_public_key.der`` - Contains a DSA 2048 bit
   key generated using OpenSSL from ``dsa_private_key.pem``.
-* ``asymmetric/PEM_Serialization/unenc-dsa-pkcs8.pem`` - Contains a DSA 1024
```

Comment:
These DSA pem paths are wrong. They are in the PKCS8 folder, not PEM_Serialization.